js_modules/dagster-ui/packages/ui-core/src/runs/AssetTagCollections.tsx

```diff
@@ -44,19 +52,26 @@ function useShowMoreDialog<T>(
   dialogTitle: string,
   items: T[] | null,
   renderItem: (item: T) => React.ReactNode,
+  sortFn?: (a: T, b: T) => number,
```

Comment:
doesn't make sense to me to pass an array and also pass a sort function for another component to use to sort your array. Just pass `items` already sorted to this component.